### ISSUE TITLE
Add load balancing

### ISSUE THREAD

**Issue by pilppe** [anderscripts/FiveM_Queue#25]
I see it as an issue since 20+ players connecting at the exact same time is not good. Since connecting causes a lot of desync when the server is just started it throws people in solo sessions. I would love a load balancing system

**Comment by anderscripts** [comment]
This can be solved by preventing connections to the server until all resources are fully loaded.  Move these lines in your server.cfg to the very bottom and you'll be fine.  

endpoint_add_tcp "0.0.0.0:30120"
endpoint_add_udp "0.0.0.0:30120"

Having them at the top, especially before the area where you start all your resources will allow people that spam direct connect to gain access to the server prior to the first resource even fully starting.  When they get in before the queue resource loads the queue resource has no idea they are already loading into the server, it expects every connection to flow through it's playerConnecting event.

**Comment by anderscripts** [comment]
Changed label to enhancement since there is a feature I have already developed for the next release that allows the server owners to configure a maximum number of players that are allowed to be in loading status consecutively with a default of five.  This is what you are asking for however the suggestion I made above is still very valid and recommended for any FiveM server.  You do not want people connecting before resources fully load on the server and this prevents that.
